src/js/Message.js

```diff
@@ -7,20 +7,19 @@ export default class {
 
     get refs() {
         return this.actionQueue
-            .map(action => {
-                return action.ref
-            })
+            .map(({ ref }) => ref)
```

Comment:
I think I'd rather see `action => action.ref`

I like the single return thing, but not sure about the param destructuring sometimes